### PR TITLE
Create a global project manager

### PR DESCRIPTION
**Pull request by ywk253100**

### CODE REVIEW

File: src/ui/proxy/interceptors.go [hunk 1]
```diff
@@ -102,7 +101,7 @@ func newPMSPolicyChecker(pm projectmanager.ProjectManager) policyChecker {
 // TODO: Get project manager with PM factory.
 func getPolicyChecker() policyChecker {
 	if config.WithAdmiral() {
-		return newPMSPolicyChecker(pms.NewProjectManager(config.AdmiralEndpoint(), ""))
+		return newPMSPolicyChecker(config.GlobalProjectMgr)
```
Comment (wy65701436) [inline src/ui/proxy/interceptors.go]: it doesn't need the condition here, right?

File: make/dev/docker-compose.yml [hunk 2]
```diff
@@ -83,6 +83,7 @@ services:
       - ./common/config/ui/private_key.pem:/etc/ui/private_key.pem:z
       - /data/secretkey:/etc/ui/key:z
       - /data/ca_download/:/etc/ui/ca/:z
+      - /data/token:/etc/ui/token:z
```
Comment (reasonerjt) [inline make/dev/docker-compose.yml]: is token a folder or a file?  need to make sure it's 600

File: src/ui/config/config.go [hunk 3]
```diff
@@ -105,9 +111,19 @@ func initProjectManager() {
 	}
 
 	// integration with admiral
-	// TODO create project manager based on pms using service account
 	log.Info("initializing the project manager based on PMS...")
-	GlobalProjectMgr = pms.NewProjectManager(AdmiralEndpoint(), "")
+	// TODO read ca/cert file and pass it to the TLS config
+	AdminserverClient := &http.Client{
```
Comment (reasonerjt) [inline src/ui/config/config.go]: Please consider increase the concurrent idle number

File: src/ui/projectmanager/pms/token.go [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+}
+
+// FileTokenReader reads token from file
+type FileTokenReader struct {
+	Path string
+}
+
+// ReadToken ...
+func (f *FileTokenReader) ReadToken() (string, error) {
+	file, err := os.Open(f.Path)
```
Comment (reasonerjt) [inline src/ui/projectmanager/pms/token.go]: There may be issue in concurrent situation that the file be updated by another process.  Need to coordinate with integration team for that.  Please open another issue to track.

**Pull request merged**